rllib/algorithms/sac/sac.py

```diff
@@ -82,6 +82,8 @@ def __init__(self, algo_class=None):
             "critic_learning_rate": 3e-4,
             "entropy_learning_rate": 3e-4,
         }
+        self.policy_lr = 3e-5
```

Comment:
Suggestions:
1) Let's add a third property `self.critic_lr` and set `self.lr` to None by default. This increases clarity and expressiveness. Otherwise, users (and us!) will forever have to open this sac.py file, just to quickly check, which one of the three lrs is the one covered by the default `self.lr`, and which 2 have their own property.
2) Add to `validate()` a quick check for a) new stack and - if yes - b) `self.lr` must be None, otherwise raise an informative error explaining that there are 3 different learning rates properties and `self.lr` should NOT be used.
3) Switch `config.lr` vs `config.critic_lr` in the respective `SACLearner` methods.